### DISCUSSION TITLE
Anzeigefehler?

### DISCUSSION THREAD

**Discussion by norbertle** [solectrus/solectrus#2935] [General]
Hallo zusammen,

ich habe das Phänom, allerdings nicht immer, das wenn ich zwischen Tagesanzeige und Top 10 wechsle bei Top 10 einen höheren Wert in der Anzeige habe als in der Tagesanzeige für die Erzeugung. Das gleicht sich aber dann wieder an.
Ist das ein Anzeige oder "Berechungsfehler"? 

Gruß

Norbert

**Comment by ledermann** [comment]
Hört sich nach einem bekannten Problem an, für das es leider noch keine Lösung gibt.
https://github.com/solectrus/solectrus/issues/1894

Das Darstellungsproblem betrifft nur den aktuellen Tag/Woche/Monat/Jahr. Mit fortschreitender Datensammlung ist es immer seltener zu sehen.
